resources/deployment.yaml

```diff
@@ -42,6 +42,11 @@ spec:
           requests:
```

Comment:
We should just let the kernel take care of resource allocation. I would remove this resource requests here. Otherwise LGTM. Please remove this and I'll LGTM